app/views/users/new.html.erb

```diff
@@ -1,7 +1,10 @@
-<h1><%= t(".title") %></h1>
+<div><%= t(".title") %></div>
 <%= simple_form_for @user do |f| %>
   <%= f.input :name %>
   <%= f.input :email %>
   <%= f.input :password %>
+  <% if current_user&.admin? %>
```

Comment:
This line `if current_user&.admin?` hides that it really is about `if current_user and current_user.admin?`
Since you already have an `admin?` helper method, I would use that instead.

Also, once more, consider creating admin namespaced controllers for this :)